packages/common/src/lib/entity/shared/entity.interfaces.ts

```diff
@@ -29,7 +29,10 @@ export interface EntityStateManagerOptions {
   store?: EntityStore<object>;
 }
 
-export interface EntityStoreStrategyOptions {}
+export interface EntityStoreStrategyOptions {
```

Comment:
These options are specific to  EntityStoreFilterCustomStateStrategy. Create a new interface instead of extending this one.